.dev/tests/cypress/helpers.js

```diff
@@ -510,13 +546,7 @@ export function hexToRGB( hex ) {
 	return 'rgb(' + +r + ', ' + +g + ', ' + +b + ')';
 }
 
-function getIframeDocument( containerClass ) {
-	return cy.get( containerClass + ' iframe' ).its( '0.contentDocument' ).should( 'exist' );
-}
-
-export function getIframeBody( containerClass ) {
-	return getIframeDocument( containerClass ).its( 'body' ).should( 'not.be.undefined' )
-		// wraps "body" DOM element to allow
-		// chaining more Cypress commands, like ".find(...)"
-		.then( cy.wrap );
+// Mailserver is usually setupped on CI
+export function MailServerSetupped() {
```

Comment:
@olafleur while I wish Setupped was a work, sadly it is not. 😆

Let's use something like `MailServerIsSetup` or `MailServerIsReady`.

Reply:
Please also see my comment, regarding Mail Server.

Reply:
Ha! Ha! Thanks, I was sure `setupped` existed ;)